src/Styleguide/Elements/Select.tsx

```diff
@@ -0,0 +1,116 @@
+import React from "react"
+import styled, { css } from "styled-components"
+import { Responsive } from "../Utils/Responsive"
+import { Sans } from "@artsy/palette"
+
+import { PositionProps, space, SpaceProps, themeGet } from "styled-system"
+
+export interface SelectProps extends PositionProps, SpaceProps {}
+
+export class Select extends React.Component<SelectProps> {
+  render() {
+    return (
+      <Responsive>
+        {({ xs }) => {
+          if (xs) return <LargeSelect />
+          else return <SmallSelect />
```

Comment:
This behavior really only happens on the sort field, right? Might want to ask Brian tomorrow, but I don't think it's a general Select behavior.

Reply:
Can you elaborate? In the spec it says to display the native select menu on click.